packages/cannon-worker-api/src/cannon-worker-api.ts

```diff
@@ -65,7 +73,7 @@ export class CannonWorkerAPI extends EventEmitter {
     quaternions: Float32Array
   }
 
-  private config: Required<CannonWorkerProps>
+  private config: Required<Omit<CannonWorkerProps, 'frictionGravity'>> & { frictionGravity?: WorldProps['frictionGravity'] }
```

Comment:
Please don't make an exception for this, we send over the config with default values specified.

Reply:
Open to ideas on expressing that this an optional value without adding this exception - I admit, it is pretty ugly!

Reply:
This is an internal interface, we don't have any optional properties here. If an acceptable value is `undefined`, that is okay.